kfs-core/src/main/java/edu/arizona/kfs/fp/document/DisbursementVoucherDocument.java

```diff
@@ -1,27 +1,205 @@
 package edu.arizona.kfs.fp.document;
 
 import org.apache.log4j.Logger;
+import org.kuali.kfs.fp.document.DisbursementVoucherConstants;
+import org.kuali.kfs.sys.businessobject.Bank;
+import org.kuali.kfs.sys.businessobject.GeneralLedgerPendingEntry;
+import org.kuali.kfs.sys.businessobject.GeneralLedgerPendingEntrySequenceHelper;
+import org.kuali.kfs.sys.businessobject.GeneralLedgerPendingEntrySourceDetail;
 import org.kuali.kfs.sys.businessobject.SourceAccountingLine;
-
 import edu.arizona.kfs.fp.businessobject.DisbursementVoucherSourceAccountingLineExtension;
+import edu.arizona.kfs.fp.businessobject.PaymentMethod;
+import edu.arizona.kfs.fp.service.PaymentMethodGeneralLedgerPendingEntryService;
+import edu.arizona.kfs.sys.KFSConstants;
+import edu.arizona.kfs.sys.KFSPropertyConstants;
+import edu.arizona.kfs.vnd.businessobject.VendorDetailExtension;
+import org.kuali.rice.krad.service.BusinessObjectService;
+import org.apache.commons.lang.StringUtils;
+import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.kfs.vnd.businessobject.VendorAddress;
+import org.kuali.kfs.vnd.businessobject.VendorDetail;
+import org.kuali.rice.kns.service.DocumentHelperService;
+import org.kuali.rice.kns.document.authorization.DocumentAuthorizer;
+import org.kuali.rice.kew.api.KewApiConstants;
+import org.kuali.rice.krad.util.GlobalVariables;
+import org.kuali.rice.kns.document.authorization.TransactionalDocumentPresentationController;
+import org.kuali.rice.krad.util.KRADConstants;
+import org.kuali.rice.krad.util.ObjectUtils;
+import org.kuali.rice.coreservice.framework.parameter.ParameterConstants.COMPONENT;
+import org.kuali.rice.kew.framework.postprocessor.DocumentRouteStatusChange;
+
 
+/**
+ * Document class override to ensure that the bank code is synchronized with the
+ * payment method code.
+ */
+// This annotation is needed to make parameter lookups work properly
+@COMPONENT( component = "DisbursementVoucher" )

```

Comment:
This annotation is not needed. The reason this was needed in UA KFS 3.0 was that the class had a non-standard name (UaDisbursementVoucherDocument.java).